frontend/catalyst/jax_tracer.py

```diff
@@ -516,14 +516,15 @@ def nested_quantum_regions(op: Operation) -> List[HybridOpRegion]:
 
 
 @debug_logger
-def trace_to_jaxpr(func, static_argnums, abstracted_axes, args, kwargs):
+def trace_to_jaxpr(func, static_argnums, abstracted_axes, pl_capture, args, kwargs):
     """Trace a Python function to JAXPR.
 
     Args:
         func: python function to be traced
         static_argnums: indices of static arguments.
         abstracted_axes: abstracted axes specification. Necessary for JAX to use dynamic tensor
             sizes.
+        pl_capture: uses pl to capture the JaxPr
```

Comment:
@rmoyard slight suggestion: let's name this `experimental_capture` or `experimental_plxpr_capture` (too long?) to indicate it won't quite yet have feature parity

Reply:
Should we just use pennylane's global configuration for this instead of adding our own?

Reply:
It is nice for the user and I think it is possible to implement. What do you think @josh146 ?
```
qml.capture.enable()

@qjit
def circuit(x):
  ...
  return ...

circuit(0.1)
```

Reply:
Good question 🤔 I'm not too opinionated here. The only argument I have for a kwarg instead of a global config is:

- it might make it easier to test (can just create two qjits with different kwargs, and no worrying about a global state between test cases)

of course, I suppose we could do _both_ (there is a kwarg that overrides the global config)

Reply:
I prefer not to mix both system, so I will go with kwarg `experimental_capture`

Reply:
But do we not have to use their global context under the hood anyways?

Reply:
Yes on our side, but I prefer users to only have one way to trigger it

Reply:
Totally agree on users only having one option, which is why I think reusing the one put in-place by pennylane makes the most sense.

Reply:
We cannot document it if we integrate it with qml.capture.enable. Also Tom and @isaacdevlugt agreed on the kwargs approach in the epic. @josh146 @isaacdevlugt Pls make a call on what approach you want :+1:

Reply:
Would a user have to have capture enabled to be able to set `experimental_capture = True`?

Reply:
Do we even want to advertise this users right now? What would be the purpose?
The only use-case I can see is internal: ongoing development and testing for the feature. 

Note that catalyst does not publicly document internals contrary to pennylane, only an explicitly selected user api.
Another downside of adding it to the public qjit api is that changing it down the line will be a breaking change (even if we tag the keyword "experimental").

Reply:
> The only use-case I can see is internal: ongoing development and testing for the feature.
That's a good point 🤔... I mean, we certainly want to _support_ a user who's interested in this (for whatever reason 😅), but we don't need to be overly concerned about the UI at this stage. It makes sense to:

- have this behaviour be well documented 
- not create a generally useless UI that incurs tech debt

Reply:
And, as a user (whoever this fictitious user might be), I would expect that if I do `qml.capture.enable_capture`, my program is being captured by _PennyLane_! It's **_qml_**.capture.enable_capture.

Reply:
As discussed before in the thread, we do not want to have both systems. @josh146 @isaacdevlugt  What makes the most sense for you?

1) Keyword argument in qjit (better control and documented)
2) Use of qml.capture (better integration and more intuitive)

Reply:
We go with 1. for the moment, solved offline